### ISSUE TITLE
New Feature: small UI improvements

### ISSUE THREAD

**Issue by distributev** [hypy13/django-daisy#1]
This project is really nice. I found it while using 

https://github.com/unfoldadmin/django-unfold and saying "this is great but I wish it had daisy UI support" and looking for an alternative I found your project and I said "just what I need"

Then I noticed the Save Buttons and the Filter always on taking unnecessary space on the right sidebar and tried to configure them to be more discrete (like in Django unfold admin) but could not find a suitable configuration for that.

Having a django admin which combines the django unfold ergonomics + daisy UI components / themes support would make the best django admin theme.

Look how discrete but still useful the "Filters" are here.

https://demo.unfoldadmin.com/en/admin/formula/driver/

You could make these configurable - user could decide how and where it wants to get the Filters and Save Buttons being 
displayed.

![change_form](https://github.com/user-attachments/assets/ef6563c2-7e64-4f9d-800c-efbf8a7b1855)
![listing](https://github.com/user-attachments/assets/1c454c71-3b95-417c-a792-c2637bdf28e4)

**Comment by hypy13** [comment]
Hi Dear,

Thank you for your thoughtful suggestion
The examples you provided are very insightful

I’ll keep you updated!

**Comment by distributev** [comment]
Thank you also, your admin theme is already very promising.

An interesting approach might be to fork the already good
https://github.com/unfoldadmin/django-unfold

and add the daisy ui plugin on top of it.

This will result in a django admin theme with good ergonomics + "Django Admin Specifics/Django admin has many dynamic forms/widgets/Unfold customizes these carefully" and, on top of it, with daisyui useful components and theming mechanism (and out of the box themes).

**Comment by hypy13** [comment]
@distributev 
Thank you, distributev! I really like Django-Unfold, and it’s a good idea to work on it. What you said is considerable, and I plan to do it.

 However, I want to focus on making my own project—something simple yet powerful—and I have plans for it. It’s like my own journey to learn a lot from you all. :heart:

**Comment by distributev** [comment]
Forking doesn't mean it is not your own, it means you have a solid foundation to start with. The open source world is full of successful open source projects forked from other initial open source projects.

see ownCloud vs Nextcloud and many other projects.

_"Since forking the ownCloud core in 2016, Nextcloud added a considerable number of features ranging from videochat to weather app to its rebranded software."_

https://owncloud.com/compare-filesharing/owncloud-vs-nextcloud/

**Comment by hypy13** [comment]
@distributev
You're right. I'll think it over and take some time to look into Django Unfold. :+1:

**Comment by hypy13** [comment]
Hi @distributev 
hope u r well, 
thanks for sharing ur ideas, and i applied your suggested reforms

may you check it out?

**Comment by distributev** [comment]
Hi @hypy13 

Your theme was already promising and at the speed at which you are adding new features I'm sure will become one of the top Django admin themes.

I'm willing to try again your theme but I would need your confirmation that the below django unfold capabilities which
I currently use in my app are also possible using your theme.

so - Using django unfold theme I build a couple of screens which are quite custom (deviating from the out of the box Django entity CRUD paradigm screens) using the below customization hooks which django unfold offers.

Even with the Django unfold documentation being available I went through couple of "hickupts" until I got everything working.

https://unfoldadmin.com/docs/styles-scripts/
https://unfoldadmin.com/docs/dashboard/
https://unfoldadmin.com/docs/pages/

Using the below code I also added my own 
menu into the left admin sidebar which unfold offers.

```
UNFOLD = {
    "SITE_TITLE": "CRM Admin",
    "SITE_HEADER": "CRM",
    "SITE_ICON": None,
    # Optional color customization - commented out to use Unfold defaults
    # "COLORS": {
    #     "primary": {
    #         "50": "250 245 255",
    #         "100": "243 232 255",
    #         # ... more color values
    #     },
    # },
    "STYLES": [
        lambda request: static("css/custom_styles.css"),  # Point to your existing file
    ],
    "SCRIPTS": [
        lambda request: static("js/custom.js"),
    ],
    "SIDEBAR": {
        "show_search": False,  # Search in applications and models names
        "show_all_applications": False,  # Dropdown with all applications and models
        "navigation": [
            {
                "title": _("Navigation"),
                "separator": False,  # Top border
                "collapsible": False,  # Collapsible group of links
                "items": [
                    {
                        "title": _("Screen1"),
                        "icon": "email",  # Choose an appropriate icon
                        "link": reverse_lazy("admin:greenfields_screen1_changelist"), 
                        "badge": None,
                        "permission": lambda request: request.user.is_superuser,
                    },
                    {
                        "title": _("Screen2"),
                        "icon": "queue",  # Choose an appropriate icon
                        "link": reverse_lazy("admin:greenfields_screen2_changelist"), 
                        "badge": None,
                        "permission": lambda request: request.user.is_superuser,
                    },
                    {
                        "title": _("Screen3"),
                        "icon": "dashboard",  # Supported icon set: https://fonts.google.com/icons
                        "link": reverse_lazy("admin:greenfields_screen3"),  # Link to your custom view
                        "badge": None,
                        "permission": lambda request: request.user.is_superuser,
                    }
                ],
            },
        ],
    },
}
```  

If you'll need additional information I can gladly share with you how I've done specific things in unfold.
Let me know.

**Comment by hypy13** [comment]
Thank you for your kind words and for reaching out!
I tried to implement same functionality also updated Readme 
for custom configurations in settings file

you can add "DAISY_SETTINGS" in your settings.py and customize some behavior of theme

**Comment by distributev** [comment]
Hello,

I the readme I cannot find how to provide my custom menu entries or how I can define my one custom .js or .css file - 

Basically how to achieve equivalents of the below customization hooks.

https://unfoldadmin.com/docs/styles-scripts/
https://unfoldadmin.com/docs/dashboard/
https://unfoldadmin.com/docs/pages/

**Comment by hypy13** [comment]
hi again :)

its not actually exactly like that
but i tried to make it

for menu overrides you can create templates/admin/parts/app_list.html 
as like native django admin

---
and for custom style or scripts you can achieve it by defining ```EXTRA_STYLES``` or  ```EXTRA_SCRIPTS``` in settings.py:
```py
DAISY_SETTINGS = {
    'SITE_TITLE': 'Django Admin',  # The title of the site 
    'SITE_HEADER': 'Administration',  # Header text displayed in the admin panel
    'INDEX_TITLE': 'Hi, welcome to your dashboard',  # The title for the index page of dashboard
    'SITE_LOGO': '/static/admin/img/daisyui-logomark.svg',  # Path to the logo image displayed in the sidebar
    'EXTRA_STYLES': [],  # List of extra stylesheets to be loaded in base.html (optional)
    'EXTRA_SCRIPTS': [],  # List of extra script URLs to be loaded in base.html (optional)
}
```

**Comment by distributev** [comment]
Besides the custom css, js and menu I also need the option to create any number of custom screens + possibility to customize the existing django dashboard screen. I am using these features from django unfold.

https://unfoldadmin.com/docs/dashboard/
https://unfoldadmin.com/docs/pages/

**Comment by hypy13** [comment]
actually everything is possible here too,
just like unfold you can override dashboard or pages

the point here is i have to make documentation for it

i will write  override documentations asap :heart:

**Comment by distributev** [comment]
great, documentation is indeed important otherwise people will not be able to use your admin dashboard.

List your package here

https://djangopackages.org/grids/g/admin-interface/
https://djangopackages.org/grids/g/themes/

so people will find your project.

**Comment by hypy13** [comment]
Sure 
Thanks for helping the project to grow up
I really appreciate it 

Anyway ill be happy for customization process in ur project if needed

**Comment by tushortz** [comment]
@hypy13 lovely theme. i just found it out today and it looks already promising. Does this support django import export?
